Specification/1._Introduction.md

```diff
@@ -29,24 +25,22 @@ The latter imposes severe restrictions on the amount of logic needed to implemen
 for common high-level functions, such as network discovery, node configuration, node firmware update,
 node status monitoring (which naturally grows into a vehicle-wide health monitoring),
 network-wide time synchronization, dynamic node ID allocation (a.k.a. plug-and-play), etc.
-* **Open specification and reference implementation** - the UAVCAN specification is open and freely available
-for anyone; the reference implementations are distributed under the terms of the
+* **Open specification and reference implementation** - the UAVCAN specification is open and freely available; the reference implementations are distributed under the terms of the
 [MIT License](http://en.wikipedia.org/wiki/MIT_License).
 
 ## Specification update and approval process
 
-UAVCAN development team is charged with advancing the specification based on the input from adopters
-gathered via the [mailing list](/Contact), where everyone can contribute to the discussion.
+The UAVCAN development team is charged with advancing the specification based on input from adopters. This feedback is gathered via the [mailing list](/Contact), which is open to everyone.
 
 The set of standard DSDL definitions is one of the cornerstone concepts of the specification.
```

Comment:
I'm modified for your review. Ideally an overview like this should not have too many off links because it makes it hard to get the whole picture if you're jumping around. That said, rules are made to be broken. IMO we could do without the forward link.